Kickstarter-iOS/Views/Controllers/ProjectUpdatesViewController.swift

```diff
@@ -5,7 +5,7 @@ import SafariServices
 
 internal final class ProjectUpdatesViewController: WebViewController {
   fileprivate let viewModel: ProjectUpdatesViewModelType = ProjectUpdatesViewModel()
-
+  fileprivate let activityIndicator = UIActivityIndicatorView()
```

Comment:
Oh one thing - I think at some point we said we'd change `fileprivate`s back to `private` as we encounter them in the codebase but no big deal!